test/clj/game_test/cards/resources.clj

```diff
@@ -2776,6 +2776,94 @@
      (is (= 2 (count (get-program state))) "2 Programs installed")
      (is (= 6 (:credit (get-runner))) "Artist discount applied new turn"))))
 
+(deftest the-class-act
+  ;; The Class Act
+  (testing "Vanilla test"
+    (do-game
+     (new-game {:runner {:deck ["The Class Act" (qty "Sure Gamble" 5) "Easy Mark" "Corporate Defector"]}})
+     (starting-hand state :runner ["The Class Act" "Corporate Defector" "Easy Mark"])
```

Comment:
This can actually be accomplished by adding `:hand ["The Class Act" "Corporate Defector" "Easy Mark"]` in the new-game runner map! I know it's functionally the same, but it does mean you don't have to list things twice, cuz it'll combine all cards in `:deck` and in `:hand` before moving the cards from deck to hand.

Reply:
To make sure I'm being clear:
```clojure
     (new-game {:runner {:deck [(qty "Sure Gamble" 5)]
                         :hand ["The Class Act" "Corporate Defector" "Easy Mark"]}})
```